src/Applications/LDAS_App/lenkf.j.template

```diff
@@ -736,8 +739,15 @@ endif
 #                 Re-Submit Job
 #######################################################################
 
-if ( $rc == 0 ) then
+if ( $COUPLING == 1) then
+    /bin/rm -f $HOMDIR/lenkf_job_completed.txt
```

Comment:
Weiyuan suggested moving this all the way up to the beginning to catch any leftover files from older runs

Reply:
moved up .